psi4/driver/p4util/python_helpers.py

```diff
@@ -118,11 +119,11 @@ def _resolve_target(key, target):
 
 
 @staticmethod
-def _core_wavefunction_build(mol, basis=None):
+def _core_wavefunction_build(mol, basis=None, *, quiet: bool = False):
```

Comment:
exactly, you can use `/` and `*` to demand positional-only and keyword-only access. It makes fns more extensible to new arguments in case some user has been doing all positional lists and looking like a dgemm call.

Reply:
Neat, thanks!